addons/pos_customer_display/static/src/app/root.xml

```diff
@@ -2,33 +2,36 @@
 <templates id="template" xml:space="preserve">
     <t t-name="pos_customer_display.CustomerDisplay">
         <t t-set="bgImgSrc" t-value="session.customer_display_bg_img ? 'url(/web/image/pos.config/' + pos.config.id + '/customer_display_bg_img)' : 'none'" />
-        <t t-set="bgColor" t-value="'#343A40'" />
-        <div class="d-flex h-100">
-            <div class="d-flex flex-column flex-grow-1">
-                <div class="d-flex justify-content-center" t-att-style="session.has_bg_img ? `background-image: url(/web/image/pos.config/${session.config_id}/customer_display_bg_img);` : `background-color: ${bgColor};`">
-                    <img t-attf-src="/logo?company={{session.company_id}}" />
+        <div class="d-flex flex-column flex-md-row-reverse h-100">
+            <div class="o_customer_display_sidebar d-flex flex-column flex-basis-md-25 align-items-center justify-content-between gap-2 p-4 bg-700" t-att-style="session.has_bg_img ? `background-image: url(/web/image/pos.config/${session.config_id}/customer_display_bg_img); min-width: 30%;` : ``">
+                <div class="o_customer_display_logo d-flex mw-50 mw-md-100 p-2 rounded-3 bg-white">
+                    <img class="img-fluid" t-attf-src="/logo?company={{session.company_id}}"/>
                 </div>
-                <div t-if="!order.finalized and order.lines?.length" class="m-2 p-3 rounded-3" t-attf-style="background-color: {{bgColor}};">
-                    <h3 class="text-white">Your Order</h3>
+                <div class="d-flex align-items-center ps-3 pe-2 py-1 rounded-3 text-bg-dark small">Powered by <OdooLogo style="'width: 3rem;'" monochrome="true"/></div>
+            </div>
+            <div class="o_customer_display_main d-flex flex-column flex-grow-1 overflow-auto">
+                <div t-if="!order.finalized and order.lines?.length" class="d-flex px-4 py-1 text-bg-dark">
+                    <h3 class="text-reset pt-2">Your Order</h3>
                 </div>
-                <OrderWidget t-if="!order.finalized" lines="order.lines" t-slot-scope="scope" style="'background-color: #f6f6f6;'">
-                    <Orderline line="scope.line" class="'bg-white mx-3 my-1 rounded-3' + (scope.line.isSelected ? ' selected' : '')"/>
+                <OrderWidget t-if="!order.finalized" lines="order.lines" t-slot-scope="scope" class="'bg-300'" style="'scroll-snap-type: y mandatory;'">
+                    <Orderline line="scope.line" imgSize="'4rem'" infoListClasses="''" class="'o_customer_display_orderline d-flex align-items-center mx-2 my-1 ps-1 pe-3 rounded-3 bg-white fs-3' + (scope.line.isSelected ? ' selected' : '')"/>
```

Comment:
I find the overall design-line too "bulky".
At 1280x720 res, we can list 7 items only, it's not enough IMO. I think that we don't need cards...
Lightspeed and squareup are good examples of how these lists can be deeply simplified.

| Current | raw test |
|--------|--------|
| ![image](https://github.com/odoo-dev/odoo/assets/8558351/b32122a0-92dc-4e43-95d7-2c707769e712)| ![image](https://github.com/odoo-dev/odoo/assets/8558351/795646a4-1ec4-4988-86e1-3d17779b0dff) |